test/Codenizer.HttpClient.Testable.Tests.Unit/WhenMatchingRoutes.cs

```diff
@@ -9,14 +13,18 @@ namespace Codenizer.HttpClient.Testable.Tests.Unit
     public static class ConfiguredRequestsExtensions
     {
         internal static RequestBuilder? Match(this ConfiguredRequests configuredRequests, HttpMethod method, string uri,
-            string? accept)
+            string? accept, AuthenticationHeaderValue? auth = null)
```

Comment:
Also a small nit: rename `auth` to `authorization`. I'd prefer real words for variables and not abbreviations 😊